.circleci/config.yml

```diff
@@ -0,0 +1,109 @@
+version: 2.1  # The version of CircleCI platform being used
+executors:
+  python-docker:
+    working_directory: ~/AutoDH 
+    docker:
+      - image: cimg/python:3.6
+
+
+jobs:  # The jobs level contains a collection of arbitrarily named children
+  run-tests:
+    executor: python-docker
+    steps:  # The steps collection is an ordered list of run directives
+      - checkout  # "checkout" checks out the source code to "working_directory"
+      - restore_cache:
+          key: pip-deps-{{ checksum "constraints.txt" }}-{{ checksum "setup.py" }}-{{ arch }}
```

Comment:
prefix this with `pip-deps-v1-`. Adding a version makes it easy to change the cache in case it becomes stale or we need to do a force update

Reply:
I did this in both spots.